test/check-application

```diff
@@ -14,41 +14,42 @@ import testlib
 
 
 class TestApplication(testlib.MachineCase):
+    def createCertificate(self, nickName):
+        m = self.machine
+
+        m.execute("selfsign-getcert request -d /etc/pki/nssdb -n {0}".format(nickName))
+
+    def testEmptyState(self):
+        b = self.browser
+        m = self.machine
+
+        m.execute("systemctl stop certmonger.service")
+
+        self.login_and_go("/certificates")
+        b.wait_text(".pf-c-empty-state h1", "certmonger.service is Not Active")
```

Comment:
Modelling the wording around that in podman, I would change this to

`Certmonger service is not active`

Note that it's sentence case, due to PF4 now [recommending that everywhere.](https://www.patternfly.org/v4/design-guidelines/content/grammar-and-terminology#capitalization)

Reply:
fixed